internal/rootcoord/root_coord.go

```diff
@@ -1657,6 +1662,86 @@ func (c *Core) AlterAlias(ctx context.Context, in *milvuspb.AlterAliasRequest) (
 	return merr.Status(nil), nil
 }
 
+// DescribeAlias describe collection alias
+func (c *Core) DescribeAlias(ctx context.Context, in *milvuspb.DescribeAliasRequest) (*milvuspb.DescribeAliasResponse, error) {
+	if code, ok := c.checkHealthy(); !ok {
+		return &milvuspb.DescribeAliasResponse{
+			Status: merr.Status(merr.WrapErrServiceNotReady(code.String())),
+		}, nil
+	}
+
+	metrics.RootCoordDDLReqCounter.WithLabelValues(DescribeAlias, metrics.TotalLabel).Inc()
+	tr := timerecord.NewTimeRecorder(DescribeAlias)
+
+	log.Ctx(ctx).Info("received request to describe alias",
+		zap.String("role", typeutil.RootCoordRole),
+		zap.String("db", in.GetDbName()),
+		zap.String("alias", in.GetAlias()))
+
+	if in.GetAlias() == "" {
+		return &milvuspb.DescribeAliasResponse{
+			Status: merr.Status(merr.WrapErrParameterMissing("alias", "no input alias")),
+		}, nil
+	}
+
+	collectionName, err := c.meta.DescribeAlias(ctx, in.GetAlias(), 0)
```

Comment:
I prefer to letting these two requests scheduled by the rootcoord instead of proxy.